proto/attestation/endorsement.proto

```diff
@@ -30,38 +28,57 @@ message TransparentReleaseEndorsement {
   // The endorsement statement for the underlying binary.
   bytes endorsement = 1;
 
-  // The endorser’s signature over the endorsement statement.
-  bytes endorsement_signature = 2;
-
   // The log entry as proof of inclusion of the endorsement statement in Rekor.
-  bytes rekor_log_entry = 3;
+  bytes rekor_log_entry = 2;
```

Comment:
This will cause the google3 import to complain about the renumbering. I would just leave it as it is, it does not really matter what the number is. Up to you though

Reply:
Can we address this as a flaw of the import? We should add the required disclaimer labels automatically/implicitly. The source of truth is here, not the import.

Reply:
It should always be possible to edit the description in the copybara CLs.